README.md

```diff
@@ -45,6 +45,63 @@ implemented and how they can be derived from new methods.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## How to add Fantasy Land compatibility to your library
+
+1. Add `fanatasy-land` package as a peer dependency. Add in your `package.json`:
+
+  ```js
+  {
+    ...
+    "peerDependencies": {
+      "fantasy-land": "*"
+    },
+    ...
+  }
+  ```
+
+2. The `fantasy-land` package exposes method names, you should use them for you Fantasy Land methods:
+
+  ```js
+  var fl = require('fanatasy-land')
+
+  // ...
+
+  MyType.prototype[fl.map] = function(fn) {
+    // Here goes implementation of map for your type...
+  }
+  ```
+
+## How to use Fantasy Land compatible library in your application
+
+1. Add library npm package, and `fanatasy-land` as your normal (or dev) dependecies:
```

Comment:
I suggest removing the reference to dev dependencies, as it won't often be correct to make fantasy-land a dev dependency.